other-metadata.rst

```diff
@@ -36,4 +36,31 @@ manager behavior which could include use of unpredictable slot, cache
 invalidation or explicit errors.
 
 
+.. index::
+   single: homepage; meaningful value
+   pair: homepage; gentoo.org
+
+HOMEPAGE value must be meaningful
+---------------------------------
+:Source: QA
+:Reference: https://archives.gentoo.org/gentoo-dev/message/83cc5bbd7bbe8bdf04dd3c3bc7f8a035
+:Reported: known bad values are reported by pkgcheck
+
+The HOMEPAGE specified for the package should either be dedicated
+to the package in question or make it easy to find dedicated
+information.  Packages must not use ``https://gentoo.org/`` or a similar
```

Comment:
I'd write https://www.gentoo.org/ here, otherwise I predict that someone will come along and claim that the "www." variant is less generic than the one without. :)

Reply:
Done.